src/Epic/index.stories.tsx

```diff
@@ -207,11 +129,9 @@ export const defaultStory = (): ReactElement | null => {
         return null;
     }
 
-    const Component = getComponentfromName(componentName);
-
     return (
         <StorybookWrapper>
-            <Component {...epicProps} />
+            <Epic {...epicProps}></Epic>
```

Comment:
I think we should be going through the component factory `BrazeMessageComponent` here to be consistent with the other stories (it's also closer to how we'll consume this in DCR).

Reply:
Done